# Ollama 一键安装启动
第一次运行选择 >> "重启内核并运行所有单元格"。安装步骤
1. 安装魔塔下载器
2. 下载 Ollama 安装包
3. 安装 Ollama
4. 启动 Ollama 服务
5. 运行 Ollama

https://ollama.com/library 查看支持的模型

## 1. 安装 ModelScope 下载器
详见 [Ollama-Linux安装](https://www.modelscope.cn/models/modelscope/ollama-linux)

In [ ]:
!pip install modelscope

## 2. 从 ModelScope 上下载 Ollama 安装包

In [ ]:
%%time
!modelscope download --model=modelscope/ollama-linux --local_dir /root/gpufree-data/ollama-linux --revision v0.5.1

## 3. 安装 ollama

In [3]:
!mkdir -p /root/gpufree-data/ollama-data && ln -s /root/gpufree-data/ollama-data /root/.ollama
!cd /root/gpufree-data/ollama-linux && chmod 777 ./ollama-modelscope-install.sh && ./ollama-modelscope-install.sh

>>> Installing Ollama to /usr/local/bin
>>> Unzipping Ollama-bundle to /usr/local...
>>> Creating ollama user...
>>> Adding ollama user to video group...
>>> Adding current user to ollama group...
>>> Creating ollama systemd service...
>>> systemd is not running
>>> NVIDIA GPU installed.
>>> The Ollama API is now available at 127.0.0.1:11434.
>>> Install complete. Run "ollama" from the command line.


## 4. 启动 Ollama 服务

In [ ]:
!ollama serve

## 5. 运行 Ollama
在终端输入 `ollama run llama3.2`

或者在控制台，选择 **自定义服务** 打开 11434 端口，用 Chatbox 等客户端连接